pyproject.toml

```diff
@@ -24,10 +24,13 @@ gradio = "^4.20.1"
 huggingface_hub = "^0.21.4"
 click = "^8.1.7"
 wandb = { version = "^0.16.4", optional = true }
+posthog = "^3.5.0"
+python-dotenv = "^1.0.1"
```

Comment:
remove? where is this being used here?

Reply:
why remove? these deps are needed

Reply:
i didn't see a `.env` file so was unsure